lib/data/source/github/github_datasource.dart

```diff
@@ -1,14 +1,18 @@
 import 'dart:convert';
 
 import 'package:pokedex/core/network.dart';
+import 'package:pokedex/data/source/github/models/item.dart';
 import 'package:pokedex/data/source/github/models/pokemon.dart';
 
 class GithubDataSource {
+  GithubDataSource(this.networkManager);
+
+  static const String itemsURL =
+      'https://gist.githubusercontent.com/AmbujaAK/1a47a19039a4203252ed834ff65ed140/raw/4058b8ee4a4747ad7bdeb63b69eb40ce15e5abe3/items.json';
```

Comment:
Please use this URL instead
https://gist.githubusercontent.com/hungps/48f4355fb1a89ddaf47f56961dc8a245/raw/pokemon-items.json

Reply:
I have changed the url and checked the items. working fine.